dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamsValidatorTest.java

```diff
@@ -308,6 +285,88 @@ void shouldFailParsingFiltersWithMissingValueAndTrailingColon() {
     assertEquals("Query item or filter is invalid: " + TEA_1_UID + ":lt:", exception.getMessage());
   }
 
+  @Test
+  void shouldParseFiltersWithFirstOperatorWhenMultipleValidOperandAreNotValid()
```

Comment:
The test name ends in `NotValid` and am wondering why we don't throw a BadRequestException 🤔